docs/conf.py

```diff
@@ -9,7 +9,9 @@
 
 # django-specific configuration
 from django.conf import settings
-settings.configure()
+
+# Removed for breaking doc building with django-guardian installed.
+# settings.configure()
```

Comment:
I added configuration to tox.ini to build the Sphinx documentation as a tox task.  Does that answer the question, or are you asking whether all of these lines are still necessary?